exercises/perfect-numbers/example.js

```diff
@@ -0,0 +1,69 @@
+
+export default class PerfectNumbers {
+
+  /**
+   * Calculate all the divisors for a given number.
+   * @param {number} number - natural number.
+   * @returns {array} array that contains the divisors for a given number NOT including the number itself.
+   */
+  getDivisors(number) {
+
+    let i;
+    let divs = new Array();
+
+    // Accepts only natura numbers greater than 1.
+    if (number <= 1) {
+      return divs;
+  	}
+
+    // 1 always divides everyone!
+    divs.push(1);
+
+    // Calculate the divisors up the the half of the number + 1
+    for (i = 2; i <= number / 2; i++) {
+
+      if (number % i === 0) {
+        divs.push(i);
+  		}
+    }
+
+    return divs;
+  }
```

Comment:
This looks good.  I was going to suggest an ES6-y approach using [generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Iterators_and_Generators) but we don't have the environment set up for that yet!